### ISSUE TITLE
List avaliable languages

### ISSUE THREAD

**Issue by MrYakobo** [martinandert/date-names#11]
More of an unsolvable issue I guess. I need to get input from the user about what language they have; Therefore I somehow need to know what languages that are supported by this package. Now, because of the way that this module defaults to English, I don't think that this feature would be possible without a breaking change. Well, I technically could `readdirSync` the `node_modules` folder, but that feels like a hack.

Just wanted to start a discussion about this. :)

**Comment by martinandert** [comment]
Hi! I'm not sure I understand from your description what the issue actually is. All supported languages are listed in the README.

**Comment by MrYakobo** [comment]
Yes, but I programmatically have to know. My program shows a list of languages, and the user selects one of them. Now, that array would have to be updated manually by me when an update rolls out from your package. I'd rather get that automatically when I update my dependencies. :package:
